.cspell.json

```diff
@@ -302,7 +302,9 @@
     "Noto",
     "Segoe",
     "kotlinx",
-    "reauthentication"
+    "reauthentication",
+    "argoproj",
```

Comment:
Why this is required? Every occurrence seems to be between backticks; i'd try to fix the `ignoreRegExpList`, instead 🤷

Reply:
I can confirm the `ignoreRegExpList` doesn't work. If I remove `reauthentication` the spellcheck fails even if the world is in triple backticks. I fixed it with a regex found here: https://github.com/streetsidesoftware/vscode-spell-checker/issues/202#issuecomment-377477473